test/pid_ros_parameters_tests.cpp

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#include <gtest/gtest.h>
+#include <gmock/gmock.h>
```

Comment:
```suggestion
#include "gmock/gmock.h"
```
please include it in the third paragraph ([other libraries' h-files](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes))